layouts/shortcodes/dbm-mysql-agent-config-examples.md

```diff
@@ -52,19 +64,19 @@ instances:
     custom_queries:
     - query: SELECT age, salary, hours_worked, name FROM hr.employees;
       columns:
-        - name: age
+        - name: mysql.custom.employee_age
           type: gauge
-        - name: salary
+        - name: mysql.custom.employee_salary
            type: gauge
-        - name: hours
+        - name: mysql.custom.employee_hours
            type: count
         - name: name
            type: tag
       tags:
         - 'table:employees'
 ```
-### Working with hosts through a remote proxy
-When connecting to a database host through a remote proxy, it can be useful to set a custom hostname. Utilize the `reported_hostname` option to override the hostname detected by the agent.
+### Working with hosts through a proxy
+If the Agent must connect through a proxy such as the [Cloud SQL Auth proxy](https://cloud.google.com/sql/docs/mysql/connect-admin-proxy), all telemetry will be tagged with the hostname of the proxy rather than the database instance. Utilize the `reported_hostname` option to set a custom override of the hostname detected by the agent.
```

Comment:
```suggestion
If the Agent must connect through a proxy such as the [Cloud SQL Auth proxy](https://cloud.google.com/sql/docs/mysql/connect-admin-proxy), all telemetry is tagged with the hostname of the proxy rather than the database instance. Utilize the `reported_hostname` option to set a custom override of the hostname detected by the Agent.
```